File: package/main/src/String/reverseString.ts
/**
 * Reverses a string
 * @param {string} char - String to reverse
 * @returns Reversed string
 * @example reverseString("Hello"); // "olleH"
 */
export const reverseString = (char: string): string => {
  return char.split("").reverse().join("");
};

File: package/umt_python/src/reverse_string.py
def reverse_string(char: str) -> str:
    """
    Reverses a string.

    Args:
        char: String to reverse.

    Returns:
        Reversed string.

    Example:
        >>> reverse_string("Hello")
        'olleH'
        >>> reverse_string("madam")
        'madam'
    """
    return char[::-1]
